pkg/test/environment.go

```diff
@@ -114,14 +115,15 @@ func NewRegionalEnvironment(ctx context.Context, env *coretest.Environment, regi
 	azureResourceGraphAPI := fake.NewAzureResourceGraphAPI(resourceGroup, virtualMachinesAPI, networkInterfacesAPI)
 	// Cache
 	kubernetesVersionCache := cache.New(azurecache.KubernetesVersionTTL, azurecache.DefaultCleanupInterval)
+	nodeImagesCache := cache.New(imagefamily.ImageExpirationInterval, imagefamily.ImageCacheCleaningInterval)
```

Comment:
Totally happy with this, but if you don't like having to use the cache consts here, you could have the `imagefamily` pkg offer a `NewDefaultCache` function. Just throwing ideas around really.

Reply:
I was considering that, but didn't want to bundle too much into this PR, and matches the other styling for now. If we decide to adjust cache handling with more defensive patterning, and reuse/helpers that I'd look at this potential.